### DISCUSSION TITLE
Can send/recv only Mavlink heartbeat (release 22.09)

### DISCUSSION THREAD

**Discussion by jiratto** [svpcom/wfb-ng#253] [General]
I installed latest image on raspi 3b+(22.09) both drone and gs side, camera work perfectly but when I connect raspi to pixhawk via uart, GS can receive only Mavlink Heartbeat message. I use all default config files from image. 

Please advise.
Thank you.

**Comment by svpcom** [comment]
if heartbeat is from autopilot then this is autopilot issue. Check that UART name and speed in `/etc/wifibroadcast.cfg` is correct. Also check that `wfb-cli gs` doesn't show `link lost` for telemetry stream

**Reply by InanixFR** [reply]
With Mavlink you have to ask which message should be sent and at what frequency
